test/docker-compose.v5.overrides.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.9'
+
+services:
+  kibana:
```

Comment:
What's the purpose of this Kibana instance?

Reply:
That's a nice thing, so maybe you can mention it in the ReadMe (i.e. how to start Kibana for local dev).